README.md

```diff
@@ -23,13 +9,15 @@ ______________________________________________________
 
 ### Abstract:
 [//]: <> (Briefly describe what you built and its features. What problem is the app solving? How does this application solve that problem?)
+ROMCOM is a web app that can make generated romance novel covers, both random and customizable. These covers can be saved to the app and later deleted.
 
 ### Installation Instructions:
 [//]: <> (What steps does a person have to take to get your app cloned down and running?)
+With the link to the GitHub Repository this app can be forked and cloned to the user's local, and then run using 'open index.html'
```

Comment:
We should probably detail this a little bit more in depth. A step by step list would look good.